pkg/controller/migmigration/migmigration_controller.go

```diff
@@ -97,7 +91,8 @@ var _ reconcile.Reconciler = &ReconcileMigMigration{}
 // ReconcileMigMigration reconciles a MigMigration object
 type ReconcileMigMigration struct {
 	client.Client
-	scheme *runtime.Scheme
+	scheme    *runtime.Scheme
+	resources *reconcileResources
```

Comment:
I'm having some second thoughts about whether this is a safe place to put the reconcileResources related to 1 particular reconciliation. Was originally thinking that each reconcile request gets a unique instance of this object, but now realizing that is probably not the case.

Reply:
Changed this, `reconcileResources` are no longer stored on `ReconcileMigMigration`